lib/schedule.js

```diff
@@ -223,7 +224,15 @@ Job.prototype.schedule = function(spec) {
   } catch (err) {
     const type = typeof spec;
     if ((type === 'string') || (type === 'number')) {
-      spec = new Date(spec);
+      // try to parse rrulestring
+      try{
+        spec = rrulestr(spec);
+      } catch (rruleErr) {
+        // if invalid rrule do nothing
+      }
+      if(typeof spec === 'string') {
```

Comment:
Why do nothing qnd not bubble up the error?

Reply:
> Essentially here I am testing to see if it a valid rrule string, if you pass a date object into the rrulestr function itll throw an error which we just want to ignore as if it is a date we just want to execute the code below

I didnt change the functionality of the previous code, just reworked it into the master branch so should functionally be the same.

Working on getting the tests working atm, never used tape before so who knows how long itll take haha